docs/Administration.rst

```diff
@@ -377,6 +380,47 @@ Hover your cursor over Site, then click:
 
 -  Click Add Site to add a new site. See :ref:`Creating a new site <CreatingNewSite>`
    for details.
+   
+   
+.. _RemovingApp:   
+   
+============================================
+Removing an application from eXo Platform 
+============================================
+
+When installing an :ref:`addon <eXo_addons>` in eXo Platform, an application appears in the platform's navigation
+and it could be added to spaces applications list.
+
+When :ref:`unistalling the addon <AddonsManagement.Uninstalling>`, it's corresponding navigation remains
+in the platform.
+
+To completely remove it, follow these steps:
+
+1-  Switch to site's edit mode by clicking on Edit -> Site -> Edit Navigation
```

Comment:
We need to indicate that for eXo platform 6.0, we need to install exo-layout-management to be able to edit site layout/navigation

Reply:
OK

Reply:
done